### ISSUE TITLE
Attributions' inference wrapper use the wrong logits

### ISSUE THREAD

**Issue by AntoninPoche** [FOR-sight-ai/interpreto#40]
## 🐛 Bug Report

In its `inference`, the `ClassificationInferenceWrapper` selects the logits with the maximum value. However, the selected logit should correspond to the initially predicted class on perturbed samples.

## Expected behavior

There should be a target parameter that either the user provides or we build it from inference on the initial samples.

I suggest passing it as a parameter to the `explain` method. If it is `None,` we compute it.
